### PR TITLE
H2O example with gunicorn and regression

### PR DESCRIPTION
**Pull request by cdraper-CS**
Separated out the work Aditya did to create an H2O regression example. This example requires Certifai 1.3.6. 

Modified the H2O german credit example to be consistent. It should still be possible to run the non-production version with Certifai 1.3.5, but the gunicorn version requires 1.3.6.

### CODE REVIEW

File: models/h2o_dai_regression_auto_insurance/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# Train and Download the Scoring Pipeline
+
+1. Clone this repository.
+
+2. Upload the dataset in this repository from [../../notebooks/datasets/auto_insurance_claims_dataset.csv](../../notebooks/datasets/auto_insurance_claims_dataset.csv) to
+H2O's Driverless AI and use it to auto-learn an ML model.
+
+3. Select 'Click for Actions' and then 'Predict'.
+
+4. Set the target column to 'outcome' and launch the experiment.
```
Comment (bjesudasan-CS) [inline models/h2o_dai_regression_auto_insurance/README.md:20]: Column name should be changed to `Total Claim Amount`

File: models/h2o_dai_regression_auto_insurance/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+```
+pip install toolkit/packages/all/*common*.zip
+pip install toolkit/packages/all/*model*.zip
+```
+
+3. Download and install the MOJO Python runtime from H2O, which is needed to run
+the Mojo from python. Follow the [instructions](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-pipeline-cpp.html#downloading-the-scoring-pipeline-runtimes) to download and install the runtime into your `daimodel` environment.
+
+4. Unzip the MOJO Scoring Pipeline that you downloaded at the end of your
+experiment, and copy `mojo.pipeline` into this folder.
```
Comment (bjesudasan-CS) [inline models/h2o_dai_regression_auto_insurance/README.md:53]: `mojo.pipeline` should be renamed to `pipeline.mojo`

This should also be changed for `h2o_dai_german_credit` usecase

File: models/h2o_dai_regression_auto_insurance/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+4. Set the target column to 'outcome' and launch the experiment.
+
+5. After completion of the Driverless AI Experiment, download the scoring pipeline that you want to scan with Certifai.
+
+  Download the [Mojo Scoring Pipeline](https://s3.amazonaws.com/artifacts.h2o.ai/releases/ai/h2o/dai/rel-1.8.5-64/docs/userguide/scoring-mojo-scoring-pipeline.html#mojo-scoring-pipeline-files).
+
+# Setup the Certifai Scanning Environment
+
+1. Follow the [Toolkit Setup](https://cognitivescale.github.io/cortex-certifai/docs/about) in the Certifai documentation to install the Certifai toolkit into a conda
```
Comment (bjesudasan-CS) [inline models/h2o_dai_regression_auto_insurance/README.md:28]: The link provided does not point specifically to the Toolkit setup. Is this intentional?

Comment (cdraper-CS) [reply models/h2o_dai_regression_auto_insurance/README.md]: Yes (reducing future broken links), but I tweaked to at least give the path in the navigation

**Review by bjesudasan-CS — APPROVED** [review]
LGTM

**Review by mchutani-CS — APPROVED** [review]
LGTM

**Pull request merged**